week1/solutions/q2.txt

```diff
@@ -0,0 +1,13 @@
+Space Complexity and Time Complexity used to analysis of algorithms.It tells about the effecieny of algorithm
+
+Time Complexity- Relation between input size and running time
+time complexity is also the number of operation done by input
+as we increase the input size the time complexity will also increase
```

Comment:
Wrong assumption and wrong output, time complexity is not directly proportional to input size

Reply:
The size of the input data directly affects the time required for processing . Generally,as the input size increases , so does the time complexity .